### ISSUE TITLE
Both choropleth shiny apps throw error when ran locally

### ISSUE THREAD

**Issue by fredderks** [cat-lord/crime_analysis#1]
Both the basic and advanced choropleth maps throw this error when ran locally in Rstudio 

`Error in rename(., LSOA11CD = lsoa) : unused argument (LSOA11CD = lsoa)`

See if you can reproduce the error or if it's something on my machine

Cheers

**Comment by rcatlord** [comment]
Hi Fred - thanks for checking out the crime analysis repo. I've run the code for both choropleth map apps and they don't throw any errors on my machine. This [issue](https://github.com/tidyverse/dplyr/issues/1388) on dplyr may be helpful because it suggests that your error message relates to the rename() function being overridden by another package. 

The apps could do with a rewrite anyway to include the tidyyverse and sf packages ;)

Henry

**Comment by fredderks** [comment]
Yeah, I stumbled onto it because I'm learning to do some leaflet choropleth maps in shiny myself, your repo has really helped me grasp some functions, thanks! You're probably right, the error is definitely on my end. I restarted a session and now it's throwing this error:

`Error in ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv,  : 
  Cannot open layer`

I really gotta clean up my R install so I'm gonna figure that out first.

Fred
